blocks/library/button/index.js

```diff
@@ -228,9 +230,16 @@ registerBlockType( 'core/button', {
 	save( { attributes } ) {
 		const { url, text, title, align, color, textColor } = attributes;
 
+		const buttonStyle = {
+			backgroundColor: color,
+			color: textColor,
+		};
+
+		const linkClass = 'wp-block-button-link';
```

Comment:
Should this be `wp-block-button__link` to better match our guidelines?

Reply:
Absolutely. I wasn't 100% sure what were the naming quidelines.

Reply:
Details about the guidelines here. Though, the use-case here is not mentioned because it's a "public" className and thus, it should not rely on the folder/file like the regular classNames we use. https://wordpress.org/gutenberg/handbook/reference/coding-guidelines/

Reply:
I updated class to `wp-block-button__link`.